src/uu/shred/src/shred.rs

```diff
@@ -33,13 +34,24 @@ pub mod options {
     pub const VERBOSE: &str = "verbose";
     pub const EXACT: &str = "exact";
     pub const ZERO: &str = "zero";
+    pub const RANDOM_SOURCE: &str = "random-source";
 
     pub mod remove {
         pub const UNLINK: &str = "unlink";
         pub const WIPE: &str = "wipe";
         pub const WIPESYNC: &str = "wipesync";
     }
 }
+struct Params {
```

Comment:
I would either rename this struct to `Options` (or something similar) or the vars of this type from `opts` to `params`.

Reply:
there is already this struct in this file:

`pub mod options {`

Reply:
At least technically it's no problem ;-)  An alternative name could be `WipeOptions`.